app/vacancy_sources/vacancy_source/source/united_learning.rb

```diff
@@ -60,9 +60,7 @@ def attributes_for(item)
       subjects: item["Subjects"].presence&.split(","),
       working_patterns: item["Working_patterns"].presence&.split(","),
       contract_type: item["Contract_type"].presence,
-      # TODO: This is coming through unexpectedly in the feed - the parameterize call can be removed
-      #       when the correct values are coming through
-      phases: [organisations_for(item).first&.readable_phase],
+      phases: phase_for(item),
```

Comment:
Don't know precisely what this comment refers to, but the feed contains the Phase now. So using it.